plan_entry_resolver.go

```diff
@@ -20,6 +20,7 @@ func (r PlanEntryResolver) Resolve(entries []packit.BuildpackPlanEntry) packit.B
 	var (
 		priorities = map[string]int{
 			"buildpack.yml":      3,
+			"project file":       2,
 			"*sproj":             2,
```

Comment:
Why do we have 2 different `version-source` values for this?

Reply:
Sorry, I should have read the summary of the change a bit closer. Why did we decide to go with `project file` instead of the name of the file? Wouldn't the name of the file be a bit clearer?

Reply:
How would we map a specific file name to the priority map?

Reply:
Could we not use globs like `*.*proj`? When we do the priority sort, we could do a glob or regex match.

I just think that showing the exact file we pulled this info from is much clearer than describing a general concept like `project file`.

Reply:
Yeah maybe that does make some sense. @fg-j would you be willing to looking into doing this?